files/katex/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
+  "name": "katex",
+  "repo": "Khan/KaTeX",
+  "files": {
+    "basePath": "bower_components/katex/dist",
+    "exclude": ["README.md"]
```

Comment:
I don't think this is right:

https://github.com/Khan/KaTeX/tree/v0.4.0

I think `/dist` would work though?